contributing_to_docs/doc_guidelines.adoc

```diff
@@ -713,6 +713,27 @@ code example 2 <2>
 <2> A note about the second example value.
 ....
 
+* If you must provide additional information on what a line of a code block
+represents and the use of callouts is impractical, you can use a description list
+to provide information about the variables in the code block. Using callouts
+might be impractical if a code block contains too many conditional statements to
+easily use numbered callouts or if the same note applies to multiple lines of the codeblock.
```

Comment:
@kalexand-rh 

How would this be used if the same note applied to multiple lines?

```
<variable_1>,<variable_2>:: A description
```

Or is it more obvious than that?

Reply:
@jboxman, I haven't seen many instances where a note applied to multiple lines that didn't involve the same variable showing up multiple times. Do you have a specific instance in mind? (There were other options in the style guide, and we can add more nuance to the guidance now if we need to.)

Reply:
@kalexand-rh, oh, I see what you mean. I don't have any specific examples in mind for when multiple variables are in play.